lane.py

```diff
@@ -2186,14 +2306,7 @@ def bibliographic_filter_clause(self, _db, qu, featured, outer_join=False):
         if self.fiction is not None:
             clauses.append(work_model.fiction==self.fiction)
 
-        if self.media:
-            clauses.append(work_model.medium.in_(self.media))
-
```

Comment:
This same code is in `WorkList.bibliographic_filter_clause`, so I don't think we need it here as well.